pkg/resource/defaultclass_test.go

```diff
@@ -252,6 +253,7 @@ func TestDefaultClassReconcile(t *testing.T) {
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			r := NewDefaultClassReconciler(tc.args.m, tc.args.of, tc.args.by, tc.args.o...)
+			fmt.Println(name)
```

Comment:
This looks like a rogue debug statement. I think you could use `t.Log` if you wanted to keep it around.

Reply:
Definitely rogue, will remove!